### PR TITLE
LPAL-475 Developer experience improvements

### PR DESCRIPTION
**Pull request by townxelliot**
## Purpose

[LPAL-475](https://opgtransform.atlassian.net/browse/LPAL-475)

Consists of various small fixes to get tests running better locally, and to correct a few issues with cypress feature files etc.

These changes were identified while working on LPAL-444 and originally formed part of that (larger) branch.

## Approach

n/a

## Learning

n/a

## Checklist

* [X] I have performed a self-review of my own code
* [X] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [X] I have added tests to prove my work
* [ ] I have added mandatory tags to terraformed resources, where possible
* [ ] If I added a package.json or composer.json, I also made sure this is included in the script in `.github/workflows/dependabot-update.yml`
* [ ] The product team have tested these changes

### CODE REVIEW

File: cypress/integration/common/fixtures.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const {
 // but only if we're not running under CI. In CI we leave it intact for 
 // the next scenario, to simulate the user journey
 After({ tags: "@CleanupFixtures" }, () => {
+    cy.log('cleaning up fixtures');
```
Comment (nickdavis2001) [inline cypress/integration/common/fixtures.js:9]: Do we still want this in the log?

File: cypress/start.sh [hunk 2]
```diff
@@ -46,9 +46,9 @@ fi
 
 
 RETVAL=$?
-echo printing $RETVAL
+echo "printing $RETVAL"
 
-if [ $RETVAL -eq 0 ]; then
+if [[ $RETVAL -eq 0 ]]; then
```
Comment (nickdavis2001) [inline cypress/start.sh:51]: did that not work before then? bash syntax is weird ;)

**Review by nickdavis2001 — COMMENTED** [review]
Just checking, the few savehandler files that were deleted? Is that definitely correct that they are no longer needed and/or will they get re-instated later?

**Comment by townxelliot** [comment]
> Just checking, the few savehandler files that were deleted? Is that definitely correct that they are no longer needed and/or will they get re-instated later?

I can't find any reference to them anywhere. We are also using redis as our save handler, so (to me) it looks like that code is unused. I will take a second look and confirm, though.

File: cypress/start.sh (hunk 2 above)
Comment (townxelliot) [reply cypress/start.sh:51]: I got an error about "unary operator" with the other syntax. I checked the above works in both bash and zsh.

File: cypress/integration/common/fixtures.js (hunk 1 above)
Comment (townxelliot) [reply cypress/integration/common/fixtures.js:9]: It doesn't hurt. It also makes it easier to see when the fixtures aren't being cleaned; without it, you just get an After > document log entry which doesn't explain where it comes from. I found it useful to see an explicit "I've cleaned up your fixtures" message.

**Pull request closed without merging**